.github/workflows/generate-summary.yaml

```diff
@@ -337,7 +337,7 @@ jobs:
               body: 'Bisecting with #${{ needs.generate-issues.outputs.new_issue_num }}'
```

Comment:
need to update actions/github-script@v6 as well https://github.com/patrick-rivos/gcc-postcommit-ci/pull/1006/files#diff-212c3469e283d0c467d86f01ac4ab899a704631cdf26e87daf5d75bb9abcf679R330

Reply:
Done